src/test/scala/com/mozilla/telemetry/pings/FrecencyUpdatePingTest.scala

```diff
@@ -10,46 +10,52 @@ class FrecencyUpdatePingTest extends FlatSpec with Matchers {
   "Frecency update ping" should "be properly deserialized" in {
     val rawPing =
       """{
-        |  "clientId": "e00d072c-db5c-42be-a159-a2e95466a7da",
-        |  "type": "frecency-update",
-        |  "payload": {
-        |    "frecency_scores": [
-        |      38223,
-        |      3933.4,
-        |      304933.3,
-        |      21
-        |    ],
-        |    "loss": 291989.21,
-        |    "model_version": 3,
-        |    "num_chars_typed": 5,
-        |    "num_suggestions_displayed": 6,
-        |    "rank_selected": 2,
-        |    "study_variation": "treatment",
-        |    "update": [
-        |      1.2,
-        |      3.2,
-        |      -3.1,
-        |      4.4,
-        |      0.5,
-        |      0.234,
-        |      -0.98,
-        |      0.33,
-        |      0.34,
-        |      0.28,
-        |      0.302,
-        |      0.4,
-        |      -0.8,
-        |      0.25,
-        |      0.9,
-        |      -0.8,
-        |      0.29,
-        |      0.42,
-        |      0.89,
-        |      0.39,
-        |      0.54,
-        |      0.78
-        |    ]}
-        |}
+       |  "clientId": "e00d072c-db5c-42be-a159-a2e95466a7da",
+       |  "type": "frecency-update",
+       |  "payload": {
+       |    "model_version": 140,
+       |    "frecency_scores": [38223, 3933.4, 304933.3, 21],
+       |    "loss": 291989.21,
+       |    "update": [
+       |    1.2,
+       |    3.2,
+       |    -3.1,
+       |    4.4,
+       |    0.5,
+       |    0.234,
+       |    -0.98,
+       |    0.33,
+       |    0.34,
+       |    0.28,
+       |    0.302,
+       |    0.4,
+       |    -0.8,
+       |    0.25,
+       |    0.9,
+       |    -0.8,
+       |    0.29,
+       |    0.42,
+       |    0.89,
+       |    0.39,
+       |    0.54,
+       |    0.78
+       |    ],
+       |    "num_suggestions_displayed": 1,
+       |    "rank_selected": 0,
+       |    "bookmark_and_history_num_suggestions_displayed": 1,
+       |    "bookmark_and_history_rank_selected": 0,
+       |    "num_key_down_events_at_selecteds_first_entry": 8,
+       |    "num_key_down_events": 14,
+       |    "time_start_interaction": 0,
+       |    "time_end_interaction": 2275,
+       |    "time_at_selecteds_first_entry": 1458,
+       |    "search_string_length": 13,
+       |    "selected_style": "autofill heuristic",
+       |    "selected_url_was_same_as_search_string": 0,
+       |    "enter_was_pressed": 1,
+       |    "study_variation": "training",
+       |    "study_addon_version": "2.1.1"
+       }}
```

Comment:
I'm definitely not a stickler for this kind of thing, but I think this indent now has an odd number of spaces